File: SFA-API/Domain/ShopItemBatch.cs
namespace Domain
{
    public class ShopItemBatch
    {
        public int Id { get; set; }
        public int ShopId { get; set; }
        public int ItemBatchId { get; set; }
        public int ItemCount { get; set; }
        public virtual Shop Shop { get; set; }
        public virtual ItemBatch ItemBatch { get; set; }
    }
}

File: SFA-Client/src/app/_models/shopItemBatch.ts
export class ShopItemBatch {
  id: number;
  shopId: number;
  shopName: string;
  itemBatchId: number;
  itemBatchName: string;

  constructor(shopItemBatch) {
    this.id = shopItemBatch.id;
    this.shopId = shopItemBatch.shopId;
    this.shopName = shopItemBatch.shopName;
    this.itemBatchId = shopItemBatch.itemBatchId;
    this.itemBatchName = shopItemBatch.itemBatchName;
  }
}
